log/context.go

```diff
@@ -71,15 +75,13 @@ func GetLogger(ctx context.Context) *logrus.Entry {
 func WithModule(ctx context.Context, module string) context.Context {
 	parent := GetModulePath(ctx)
 
-	if parent != "" {
```

Comment:
@dperny why is this check not needed anymore?

Reply:
Leave this check in place.